### ISSUE TITLE
Alumni, Athletics, Give need adjusting

### ISSUE THREAD

**Issue by jcorkill** [neilerdwien/homepage-2017#35]
They should be a tinge smaller and centered against the other nav text.

**Comment by neilerdwien** [comment]
The exact placement changes when the font changes.  I'd still like to explore fonts to see if we can use the real Myriad Pro.

**Comment by neilerdwien** [comment]
Adjusted.  But I learned something that affects the overall layout.  We switched from a 960-pixel wide layout to a 1200-pixel wide layout.  So I overrode some 960px values in the stylesheet to 1200px.  But there is a difference I didn't realize until now.  The 960 included 1em or so of left and right padding, so the wordmark and search box didn't touch the edge of the screen on 960px wide displays or smaller.

In your mockup, you used 1200px edge-to-edge with no padding.  I expanded the menu so it matches the mockup, but that means it will touch the edge on <= 1200 pixel screens.

I presume it would be better to include the padding and make the menu font a touch smaller.  That'll make the left edges of the wordmark and menu line up again too.

Your thoughts?
